.hadolint.yaml

```diff
@@ -17,4 +17,6 @@ override:
     # warning: Use WORKDIR to switch to a directory
     # Reason: Sometimes we don't want to change the workdir
     - DL3003
+    # Do not use --platform= with FROM. https://github.com/hadolint/hadolint/wiki/DL3029  
```

Comment:
You don't seem to do what the wiki page says. Are you sure this override is needed?

Reply:
I mean, I don't see you doing any `FROM --platform` anywhere.

Reply:
Great catch thanks! I have to do it for the framwork image, in one of my refactorings I must have deleted it, this is what gave me so many headaches for RPi :P